requirements.txt

```diff
@@ -29,10 +29,14 @@ scipy==1.9.3
 setuptools-lint
 sphinx
 sqlalchemy==1.4.43
-sumy==0.11.0
-tiktoken==0.4.0
+tiktoken==0.3.3
 torch==2.0.0
 tqdm==4.64.0
+numpy~=1.24.3
```

Comment:
Can you please clarify why this addition is necessary?

Reply:
removed in [e30538b](https://github.com/MLDSAI/OpenAdapt/pull/195/commits/e30538be1cff1f966c935f23f8a107d0b747db8e)